src/main/java/frc/robot/subsystems/IndexerSubsystem.java

```diff
@@ -16,9 +24,25 @@ public class IndexerSubsystem implements Subsystem {
 
   private final TalonFX beltMotor = new TalonFX(DEVICE_ID_BELT);
   private final VelocityTorqueCurrentFOC beltControl = new VelocityTorqueCurrentFOC(0.0);
+  private final TorqueCurrentFOC indexerSysIdControl = new TorqueCurrentFOC(0.0);
 
-  public IndexerSubsystem() {
+  private final SysIdRoutine IndexerSysIdRoutine = new SysIdRoutine(
+      new SysIdRoutine.Config(
+          null, // Use default ramp rate (1 V/s)
+          null, // Use dynamic voltage of 7 V
+          null, // Use default timeout (10 s)
+          // Log state with SignalLogger class
+          state -> SignalLogger.writeString("Indexer Sys ID", state.toString())),
+      new SysIdRoutine.Mechanism(
+          (amps) -> beltMotor.setControl(indexerSysIdControl.withOutput(amps.in(Volts))),
+          null,
+          null));
 
+  public IndexerSubsystem() {
+    var indexerTalonConfig = new TalonFXConfiguration();
+    indexerTalonConfig.MotorOutput.NeutralMode = NeutralModeValue.Brake;
+    indexerTalonConfig.Slot0 = Slot0Configs.from(SLOT_CONFIGS);
```

Comment:
Let's add current limits. I think all mechanisms should have current/torque limits from the start this year. Last year we burned out a Falcon on the intake at week 0.